examples/demo-rollup/Cargo.toml

```diff
@@ -10,6 +10,8 @@ publish = false
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+blake3 = "1.3"
+curl = "0.4"
```

Comment:
These should probably be dev-dependencies if we're only using them for tests. That will reduce the binary size and help mitigate the security risks if one of them is buggy.